doc/book/connectors/__python.rst

```diff
@@ -39,8 +39,181 @@ For an example of using Python API with
 `queue managers for Tarantool <https://github.com/tarantool/queue>`__, see
 `queue-python <https://github.com/tarantool/queue-python>`__ project at GitHub.
 
-Also there are several community-driven Python connectors:
+Also there are several community-driven Python connectors (see table below with
+feature comparison):
 
 * `asynctnt <https://github.com/igorcoding/asynctnt>`__ with asyncio support
 * `aiotarantool <https://github.com/shveenkov/aiotarantool>`__ also with asyncio support
 * `gtarantool <https://github.com/shveenkov/gtarantool>`__ with gevent support **no active maintenance**
+
+Feature comparison
+------------------
```

Comment:
I would leave a reason why aiotarantool is not in the table.

Reply:
Done.

```diff
+++ b/doc/book/connectors/__python.rst
@@ -39,17 +39,20 @@ For an example of using Python API with
 `queue managers for Tarantool <https://github.com/tarantool/queue>`__, see
 `queue-python <https://github.com/tarantool/queue-python>`__ project at GitHub.
 
-Also there are several community-driven Python connectors (see table below with
-feature comparison):
+Also there are several community-driven Python connectors:
 
 * `asynctnt <https://github.com/igorcoding/asynctnt>`__ with asyncio support
 * `aiotarantool <https://github.com/shveenkov/aiotarantool>`__ also with asyncio support
 * `gtarantool <https://github.com/shveenkov/gtarantool>`__ with gevent support **no active maintenance**
 
+Table below contains a feature comparison for asynctnt, gtarantool and
+tarantool-python. aiotarantool is absent there because it is quite outdated and
+unmaintained.
+
 Feature comparison
 ------------------
```